addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -4343,6 +4338,22 @@ const actions = {
         await dispatch('_fetchPartnerImStatus');
         dispatch('_loopFetchPartnerImStatus');
     },
+    /**
+     * @private
+     * @param {Object} param0
+     * @param {Object} param0.state
+     * @param {string} threadCacheLocalId
+     */
+    async _setThreadCacheLoading({ state }, threadCacheLocalId) {
+        const threadCache = state.threadCaches[threadCacheLocalId];
+        threadCache.isLoaded = false;
```

Comment:
This looks wrong.
A thread cache is initially not loaded, but then when it has 30 messages and a "load more" action is triggered, it should still be loaded but loading more.

I get conceptually this is strange that code in thread cache uses mutually exclusive booleans (a thread cache can't really be is loaded, loading, and loading more at the same time), and this could be replaced by a "state" (like a string, and it uses getter. Note that the reason it hasn't been done that way in the 1st place was to avoid typo with strings, and getters are not automatically observed by `useStore`...).

If I get this right, because `isLoaded` is set to `false`, template of thread during a loading more scenario could (and would most of the time in practice) re-render with no messages, and then display all messages. This would definitely flicker a lot, and would definitely mess up scroll bar adjustment.